src/VisualStudio/CSharp/Test/Interactive/Commands/InteractiveCommandHandlerTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Roslyn.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Interactive.Commands
+{
+    internal class InteractiveCommandHandlerTests
+    {
+        private string ExampleCode1 =
+@"var x = 1;
+Task.Run(() => { return 1; });";
+
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.Interactive)]
+        public void TestExecuteInInteractive()
+        {
+            // Tests that the command is unavailable without a selection.
+            AssertUnavailableExecuteInInteractive("$$");
+            AssertUnavailableExecuteInInteractive($"{ExampleCode1}$$");
+
+            // Tests cases when code is selected and submission buffer is empty.
```

Comment:
These comments suggest that the each of these blocks is testing distinct functionality.  Would it make sense to separate each block into its own test so that they can pass/fail independently?